### DISCUSSION TITLE
Need info on how to report bugs

### DISCUSSION THREAD

**Discussion by benjamin051000** [flxzt/rnote#381] [Q&A] [answered]
Experiencing hanging when I open a note. How can I get an error log? I can't seem to find any info in the repository about how to do this... I'll search the issues page next.

**Comment by benjamin051000** [comment]
Okay. I see an easy way is to run the app from the command line via `flatpak run com.github.flxzt.rnote`. This should be documented somewhere if it isn't already, along with any other stuff that may help

**Reply by flxzt** [reply]
What also might be helpful: more verbose logs by setting the debug level with `RUST_LOG=rnote=debug flatpak run com.github.flxzt.rnote` . I'll add this to the issue template

**Reply by benjamin051000** [reply]
That would be perfect, thank you

**Comment by benjamin051000** [comment, marked as answer]
Okay, I see that first one in the issue template. That seems good for now.
